docs/core/testing/unit-testing-with-dotnet-test.md

```diff
@@ -8,101 +8,126 @@ ms.custom: "seodec18"
+  ```dotnetcli
+  dotnet sln add ./PrimeService.Tests/PrimeService.Tests.csproj
+  ```
 
-```dotnetcli
-dotnet add reference ../PrimeService/PrimeService.csproj
-```
+* Add the `PrimeService` class library as a dependency to the *PrimeService.Tests* project:
 
-You can see the entire file in the [samples repository](https://github.com/dotnet/samples/blob/master/core/getting-started/unit-testing-using-dotnet-test/PrimeService.Tests/PrimeService.Tests.csproj) on GitHub.
+  ```dotnetcli
+  dotnet add ./PrimeService.Tests/PrimeService.Tests.csproj reference ./PrimeService/PrimeService.csproj  
+  ```
 
-The following shows the final solution layout:
+<a name="create-test-cmd"></a>
 
-```
-/unit-testing-using-dotnet-test
-    unit-testing-using-dotnet-test.sln
-    /PrimeService
-        Source Files
-        PrimeService.csproj
-    /PrimeService.Tests
-        Test Source Files
-        PrimeServiceTests.csproj
-```
+### Commands to create test solution
 
-To add the test project to the solution, run the [dotnet sln](../tools/dotnet-sln.md) command in the *unit-testing-using-dotnet-test* directory:
+This section contains all the commands in the previous section. Skip this section if you've created the test solution.
+
+The following commands create the test solution on a windows machine. For MacOS and unix, update the `ren` command to the OS version of `ren` to rename a file:
 
 ```dotnetcli
+dotnet new sln -o unit-testing-using-dotnet-test
+cd unit-testing-using-dotnet-test
+dotnet new classlib -o PrimeService
+ren .\PrimeService\Class1.cs PrimeService.cs
+dotnet sln add ./PrimeService/PrimeService.csproj
+dotnet new xunit -o PrimeService.Tests
+dotnet add ./PrimeService.Tests/PrimeService.Tests.csproj reference ./PrimeService/PrimeService.csproj
 dotnet sln add ./PrimeService.Tests/PrimeService.Tests.csproj
 ```
 
-## Creating the first test
+Follow the instructions for "Replace the code in *PrimeService.cs* with the following code" in the previous section.
+
+## Create a test
+
+A popular approach in test driven development (TDD) is to write a test before implementing the target code. This tutorial uses the TDD approach. The `IsPrime` method is callable, but not implemented. A test call to `IsPrime` fails. With TDD, a test is written that is known to fail. The target code is updated to make the test pass. You keep repeating this approach, writing a failing test and then updating the target code to pass.
 
-You write one failing test, make it pass, then repeat the process. Remove *UnitTest1.cs* from the *PrimeService.Tests* directory and create a new C# file named *PrimeService_IsPrimeShould.cs*. Add the following code:
+Update the *PrimeService.Tests* project:
+
+* Delete *PrimeService.Tests/UnitTest1.cs*.
+* Create a *PrimeService.Tests/PrimeService_IsPrimeShould.cs*  file.
+* Replace the code in *PrimeService_IsPrimeShould.cs* with the following code:
 
 ```csharp
```

Comment:
This code should be indented to be under the list item.

Reply:
The doc build looks much better if it's not, ditto for the information after the block.